.github/workflows/robustness-template.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: install-lazyfs
         if: ${{ inputs.lazyfsEnabled }}
         run: |
-          sudo apt update && sudo apt-get --yes install cmake libfuse3-dev libfuse3-3 fuse3
+          sudo apt -o APT::Update::Error-Mode=any update && sudo apt-get --yes install cmake libfuse3-dev libfuse3-3 fuse3
```

Comment:
Suggestion to use `apt-get` instead of `apt`, as `apt` shows warning: 
> WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

And quoting [`apt`'s man page](https://manpages.ubuntu.com/manpages/xenial/man8/apt.8.html#description):
> apt provides a high-level commandline interface for the package management system. It is
intended as an end user interface and enables some options better suited for interactive
usage by default compared to more specialized APT tools like apt-get(8) and apt-cache(8).

Reply:
Thanks for your mentioning @ivanvc , I also think  `apt-get` is better than `apt `